lib/src/validator/size.dart

```diff
@@ -21,8 +24,21 @@ class SizeValidator extends Validator {
     return packageSize.then((size) {
       if (size <= _MAX_SIZE) return;
       var sizeInMb = (size / math.pow(2, 20)).toStringAsPrecision(4);
-      errors.add("Your package is $sizeInMb MB. Hosted packages must be "
-          "smaller than 100 MB.");
+      // Current implementation of Package.listFiles skips hidden files
+      var ignoreExists = fileExists(join(entrypoint.root.dir, '.gitignore'));
+
+      var error = new StringBuffer(
+          "Your package is $sizeInMb MB. Hosted packages must be smaller than 100 MB.");
```

Comment:
A lot of these strings are longer than the 80-character line length. Unfortunately, the Dart formatter will only change whitespace, so you'll have to manually split the strings up to get them to fit.

Reply:
Will fix.